### PR TITLE
Added solution for HW_5.1

### PR DESCRIPTION
**Pull request by polinakulyk**

### CODE REVIEW

File: src/main/java/core/basesyntax/Engine.java [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+    public Engine(String type, String version, String producer, String fuel, double volume) {
+        this.type = type;
+        this.version = version;
+        this.producer = producer;
+        this.fuel = fuel;
+        this.volume = volume;
+    }
+
+    @Override
+    protected Engine clone() throws CloneNotSupportedException {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Engine.java:19]: can we remove throws?

File: src/main/java/core/basesyntax/Car.java [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+    public Car(Engine engine, String color, int year, String country, double price) {
+        this.engine = engine;
+        this.color = color;
+        this.year = year;
+        this.country = country;
+        this.price = price;
+    }
+
+    @Override
+    protected Car clone() throws CloneNotSupportedException {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/Car.java:19]: same

File: src/main/java/core/basesyntax/Engine.java (hunk 1 above)
Comment (polinakulyk) [reply src/main/java/core/basesyntax/Engine.java]: done

File: src/main/java/core/basesyntax/Car.java (hunk 2 above)
Comment (polinakulyk) [reply src/main/java/core/basesyntax/Car.java]: thank you, done

**Pull request closed without merging**